### ISSUE TITLE
[Rogue][Spell] - Cheat Death

### ISSUE THREAD

**Issue by roguebugs** [SunwellTracker/issues#1431]
Decription: Cheat Death procs even tho you are shielded.

How it works: For example you have Power word:Shield on with 1k hp and when you get hit for 1k hp(not breaking shield) it still procs the cheat death.

How it should work: Should not proc cheat death if the rog isnt dropping to actually 0% hp

Source (you should point out proofs of your report, please give us some source): https://wowgaming.altervista.org/aowow/?spell=31230

**Comment by danirzv** [comment]
it doesn't work!!! its just effect
Let me explain it more:

How it works:
when you are going to die with an attack this passive gonna get trigerred and you see the effect and wont die in that attack, but next attack gonna kill you.

How it Should work:
when you are going to die with an attack this passive gonna get trigerred and you see tge effect and wont die in that attack, **You become healed up to 10% of your Total HP and in 3 next secounds you will not recieve 90% of damages in any kind**


its to annoying bug, please fix it ASAP

**Comment by Azathothh** [comment]
that's not true danirzv - please read talent description again, cause you wrote something different in "how it should work"

![obraz](https://user-images.githubusercontent.com/37483678/62369022-91c15380-b52e-11e9-8895-3847648fd981.png)
^ 14:02:12 hit which should kill me but it didn't, next hit doesn't have reduced damage BECAUSE i have 0 resilience

![obraz](https://user-images.githubusercontent.com/37483678/62369288-7145c900-b52f-11e9-80ac-25f98707e6b2.png)
^ here i have 596resilience so damage after cheat death WAS reduced

https://www.wowhead.com/forums&topic=75577
